autoscheduler/scraper/views.py

```diff
@@ -1,6 +1,13 @@
+from collections import OrderedDict
 from rest_framework import generics
+from rest_framework.response import Response
 from .serializers import CourseSerializer
+from .serializers import SectionSerializer
+from .serializers import TermSerializer
+from .serializers import CourseSearchSerializer
 from .models.course import Course
+from .models.section import Section
+from .models.department import Department

```

Comment:
It won't let me drag it, but change lines 8-10 to
```suggestion
from scraper.models import Course, Section, Department
```

Just by the way, the only reason we can do all of these on one line and not have to do them on separate lines (like we do with serializers) is due to the contents of [`scraper/models/__init__.py`](https://github.com/aggie-coding-club/Rev-Registration/blob/master/autoscheduler/scraper/models/__init__.py) (which is required for Django models)